sqlmesh/dbt/model.py

```diff
@@ -263,7 +264,7 @@ def _big_query_partition_by_expr(self) -> exp.Expression:
 
     @property
     def sqlmesh_config_fields(self) -> t.Set[str]:
-        return super().sqlmesh_config_fields | {"cron", "interval_unit"}
+        return super().sqlmesh_config_fields | {"cron", "interval_unit", "allow_partials"}
```

Comment:
@crericha can you please confirm that this is sufficient change to enable propagation of this attribute from dbt models?